### ISSUE TITLE
proxy

### ISSUE THREAD

**Issue by CrismaruIlie** [xTACTIXzZ/Hydrazine#6]
hi, the proxies doesn't work, can you compile the latest version to binaries ? thanks

**Comment by stenocyb** [comment]
Hi, I have noticed that authentication proxies (-ap) might not work correctly, I have to take a deeper look into this issue, however after extensive testing of the '-sp' parameter, using **socks5** proxies (other types will not work) directly (host:port) and indirectly (using a file), I came to the conclusion that there is no issue. Your proxies must be either the wrong type or slow as hell. Or you might be trying to connect to a server that blocks proxies. The binaries are by the way already updated.
https://github.com/xTACTIXzZ/Hydrazine/releases
